crates/astria-composer/src/searcher/executor.rs

```diff
@@ -407,11 +466,420 @@ impl Future for SubmitFut {
 }
 
 fn sha256(data: &[u8]) -> [u8; 32] {
-    use sha2::{
-        Digest as _,
-        Sha256,
+    use sha2::Sha256;
+    Sha256::digest(data)
+}
+
+#[cfg(test)]
```

Comment:
Thank you for going through the effort of adding these tests. I am wondering if we are overspecifying these